lib/perl/Genome/Model/Tools/BedTools.pm

```diff
@@ -49,6 +49,19 @@ my %BEDTOOLS_VERSIONS = (
     '2.3.2' => Genome::Config::get('sw') . '/bedtools/BEDTools-2.3.2',
 );
 
+sub available_bedtools_versions {
```

Comment:
Would using something like `Perl::Version` (which I think we already have installed) make this sort a little more robust?